modules/create-object-collections.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operating/create-use-collections.adoc
+:_content-type: PROCEDURE
+[id="creating-object-collections_{context}"]
+= Creating deployment collections
+
```

Comment:
This procedure seems disconnected from the bullet item above. Could there be a short sentence here that describes the purpose of this procedure? Something like: "When creating a collection, you must name it and define the rules for the collection."